### PR TITLE
Save Our Ship 2 Patch Update

### PR DESCRIPTION
**Pull request by Breadbox2k19**
Standardized SOS2 apparel with its CE counterpart.
Added Ammo to SOS2 space crews.

## Testing

Check tests you have performed:
- [ x ] Compiles without warnings
- [ x ] Game runs without errors
- [ x ] (For compatibility patches) ...with and without patched mod loaded
- [ x ] Playtested a colony (specify how long)

### CODE REVIEW

**Comment by MadaraUchiha** [comment]
This doesn't fix the torpedo problem, right?

**Comment by Breadbox2k19** [comment]
> This doesn't fix the torpedo problem, right?

No, it's purely an apparel and pawn ammo patch.

File: Patches/Save Our Ship 2/PawnKindsDefs_SOS2.xml [hunk 1]
```diff
@@ -5,17 +5,122 @@
...
+        <max>900</max>
+      </weaponMoney>
+    </value>
+			</li>
+
+		<li Class="PatchOperationAddModExtension">
+  	<xpath>Defs/PawnKindDef[defName="SpaceCrewMarine"]</xpath>
+		<value>
+		  <li Class="CombatExtended.LoadoutPropertiesExtension">
+			<primaryMagazineCount>
```
Comment (N7Huntsman) [inline Patches/Save Our Ship 2/PawnKindsDefs_SOS2.xml:71]: Could use a bit more, probably 4 - 6.

File: Patches/Save Our Ship 2/PawnKindsDefs_SOS2.xml [hunk 2]
```diff
@@ -5,17 +5,122 @@
...
+        <max>1300</max>
+      </weaponMoney>
+    </value>
+			</li>
+
+		<li Class="PatchOperationAddModExtension">
+  	<xpath>Defs/PawnKindDef[defName="SpaceCrewMarineHeavy"]</xpath>
+		<value>
+		  <li Class="CombatExtended.LoadoutPropertiesExtension">
+			<primaryMagazineCount>
```
Comment (N7Huntsman) [inline Patches/Save Our Ship 2/PawnKindsDefs_SOS2.xml:93]: Could use slightly more, probably 3 - 5.

File: Patches/Save Our Ship 2/PawnKindsDefs_SOS2.xml [hunk 3]
```diff
@@ -5,17 +5,122 @@
...
+        <max>300</max>
+      </weaponMoney>
+    </value>
+			</li>
+
+		<li Class="PatchOperationAddModExtension">
+  	<xpath>Defs/PawnKindDef[defName="SpaceCrew"]</xpath>
+		<value>
+		  <li Class="CombatExtended.LoadoutPropertiesExtension">
+			<primaryMagazineCount>
```
Comment (N7Huntsman) [inline Patches/Save Our Ship 2/PawnKindsDefs_SOS2.xml:49]: Increase to 2 - 3, please.

**Comment by Breadbox2k19** [comment]
The lowish ammo is a deliberate choice, due to the close-quarter nature of boarding action, pawns are unlikely to expand much ammo(before dying) and I don't personally want players to be swamped in looted ammo.

**Comment by Breadbox2k19** [comment]
> Looks good.

The recent update have caused some new errors in the patch, please refrain from merging atm.

**Comment by N7Huntsman** [comment]
Understood. If you'd like, you can mark this PR as a draft until you're comfortable it's ready.

**Comment by Breadbox2k19** [comment]
Tested and there are no additional problems, ready to be merged.
On another note, are there issues with the Witcher patch?

**Pull request merged**